src/utils/nftRewards.ts

```diff
@@ -181,3 +181,21 @@ export async function uploadNftCollectionMetadataToIPFS({
   })
   return res.data.IpfsHash as string
 }
+
+// Determines if two NFT reward tiers are equal
+export function tiersEqual({
```

Comment:
It's at times like this, I actually miss class based languages with equality operators :-(